src/Symfony/Component/FeatureToggle/README.md

```diff
@@ -0,0 +1,75 @@
+ToggleFeature Component
+=======================
+
+The ToggleFeature component provides TODO
+
+**This Component is experimental**.
```

Comment:
Experimental components are forbidden in LTS versions, so this PR must target the 7.0 branch instead.

Reply:
done